media/css/mozorg/contribute/contribute-base.less

```diff
@@ -1036,6 +1036,10 @@ textarea:focus {
     .event-name {
         width: 40%;
     }
+
+    address {
```

Comment:
This looks great :D one small tweak, we keep the css elements in alphabetical order. To keep with that style lets put the address selector above the `td, th { ...}` selector.

Reply:
Sure.